src/erlcloud_aws.erl

```diff
@@ -146,7 +146,10 @@ aws_region_from_host(Host) ->
         [_, Value, _, _ | _Rest] ->
             Value;
         _ ->
-            application:get_env(erlcloud, aws_region, "us-east-1")
+            case default_config_get("AWS_REGION", aws_region) of
```

Comment:
I think it it worth adding `default_config_get/3` which returns default value and have just one line here
```
default_config_get("AWS_REGION", aws_region, "us-east-1")
```